airflow/bin/cli.py

```diff
@@ -28,6 +28,12 @@
 from airflow.utils.state import State
 from airflow.exceptions import AirflowException
 
+# make this configurable
+from airflow.api.client.local_client import LocalClient
+from airflow.api.client.api_client import ApiClient
+
+#client = LocalClient()
```

Comment:
Why is the LocalClient commented out? Is it either/or for the LocalClient and APIClient?